src/components/AppSidebar/AppSidebar.vue

```diff
@@ -314,6 +370,20 @@ export default {
 			this.$emit('update:starred', this.isStarred)
 		},
 
+		editTitle(event) {
+			// Don't edit the title if editing is disabled or if we clicked on a link
+			if (event.target.tagName === 'A') {
+				return
+			}
```

Comment:
Maybe you can use the `.exact` event specifier ?

Reply:
Good idea to use a specifier :+1. Works well.